apps/account-worker/src/transaction_publisher/publisher.service.ts

```diff
@@ -196,6 +204,34 @@ export class TransactionPublisherService extends BaseConsumer implements OnAppli
     }
   }
 
+  async processProxyTxn(
+    ext: any,
+    accountId: string,
+    signer: Signer,
+    signerPayload: SignerPayloadRaw,
+  ): Promise<HexString> {
```

Comment:
This should return both the signed & submitted extrinsic itself, as well as the hash, like `[SubmittableExtrinsic<'promise'>, HexString]` (see `processSingleTxn` and `processBatchTxn`). This will allow us to track the extrinsic to completion using its mortality era.